tests/modules/module-with-transform-schema.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+module.exports = async function plugin(fastify) {
+  fastify.get('/', {
+    schema: {
+      querystring: {
+        label: { type: 'string' },
+      },
+    },
+  }, function returnConfig(request, reply) {
+    reply.send({ })
+  })
+}
+
+module.exports.transformSchemaForSwagger = (schema) => {
```

Comment:
This means that the `transformSchemaForSwagger` function is the same for all the API schemas exposed by the service. Shouldn't this be defined per api instead of being for all apis?

Reply:
yes, it's equals for all schemas.
It's not possible to do it based on `api` due to `fastify-swagger`: https://github.com/fastify/fastify-swagger   `transform` function